science/exe.py

```diff
@@ -245,90 +267,335 @@ def parse(cls, value: str) -> AppInfo:
     value: str
 
 
-@_main.command()
-@click.argument("config", type=click.File("rb"), default="lift.toml")
+@dataclass(frozen=True)
+class LiftConfig:
+    file_mappings: tuple[FileMapping, ...] = ()
+    invert_lazy_ids: frozenset[str] = frozenset()
+    include_provenance: bool = False
+    app_info: tuple[AppInfo, ...] = ()
+    app_name: str | None = None
+
+
+pass_lift = click.make_pass_decorator(LiftConfig)
+
+
+@_main.group(cls=DYMGroup, name="lift")
 @click.option(
     "--file",
     "file_mappings",
+    metavar="NAME=LOCATION",
     type=FileMapping.parse,
     multiple=True,
     default=[],
     envvar="SCIENCE_EXPORT_FILE",
+    help=dedent(
+        """\
+        Map paths to files defined in your manifest.
+
+        Science looks fore each non-lazy file you define at the path denoted by its name relative
+        to the CWD you invoke science from. If any file is not at that path, you can tell science
+        to look elsewhere with: `--file <name>=<location>`.
+
+        For example, for this manifest snippet:
+
+        \b
```

Comment:
N.B.: This is a documented click trick to force the following paragraph to be rendered without re-wrapping. Used here and elsewhere for code blocks.